vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -764,47 +719,3 @@ func verifyAddPermissionlessDelegatorTx(
 
 	return nil
 }
-
-type addDelegatorRules struct {
```

Comment:
moved to staker_tx_verification_helpers.go file. These rules won't be used by stakers verification methods only